pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -539,7 +539,7 @@ func (s *Service) DeleteLoadBalancerMember(openStackCluster *infrav1.OpenStackCl
 			return err
 		}
```

Comment:
@dulek Not related this this PR, but this looks pretty janky. Does this need an issue to track it?

Reply:
We should probably just make a call and ignore 404's? Is this what you mean here?